nemo/collections/asr/parts/submodules/conformer_modules.py

```diff
@@ -207,28 +207,15 @@ def __init__(
             in_channels=d_model, out_channels=d_model * 2, kernel_size=1, stride=1, padding=0, bias=True
         )
 
-        if conv_context_size is None or (
```

Comment:
What case was this check for before ?

Reply:
It was to decide if we should use the conv1d class or the wrapper class of causalconv1d. Now we just use the causal version to support caching even for models trained with full context.

Reply:
Oh ok